frontend/snk_web/src/service/EnderecoService.js

```diff
@@ -0,0 +1,32 @@
+import Http from '../config/http'; // não precisa de .js
+
+/**
+ * Busca endereço a partir do id do cliente.
+ *
+ * @param {number} idCliente
+ *
+ * @returns {Promise}
+ */
+ const getEnderecoByIdCliente = async (idCliente) => {
```

Comment:
Como estamos filtrando Endereços por algo diferente do `id` dele, acho que vale renomearmos pra `list` ao invés de `get` aqui. `get` é mais voltado pra pegar 1 único registro, aqui vamos trazer múltiplos.
```suggestion
 const listEnderecoByIdCliente = async (idCliente) => {
```

Reply:
Neste caso, é necessário fazer a seguinte mudança também?:
`const listEnderecoByIdCliente = async (idCliente) => {
    return Http.get(`/endereco?idCliente=${idCliente}`);
  };` => 

`const listEnderecoByIdCliente = async (idCliente) => {
    return Http.list(`/endereco?idCliente=${idCliente}`);
  };`

Reply:
Não, seria só no nome da função mesmo, o verbo da requisição HTTP continua sendo o GET ;)